specs/0117-email-opt-in-unsubscribe.md

```diff
@@ -0,0 +1,109 @@
+
+They should be shown two buttons, one to opt in to emails or one declining to opt in. This should not be shown again if the user clicks either of the buttons, but the standalone page for email consent (above) should be shown again upon next login if the user closed their tab/browser.
+
+(Section: content)
+- Header: `Welcome to Pontoon!`
+- Sub-header: `You have successfully created your Pontoon account.`
+- Body: `Would you like to receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform? This can be changed at any time in your settings.`
+- Button: `Enable email updates`
+- Button: `No, thank you`
+- Consent: `By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`
+
+## Additional card for tour
+
+As part of new user onboarding, point new users to user preferences and call out the ability to enable email communication preferences.
+
+(Section: content)
+- Header: `Stay up to date`
+- Body: `You can receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform by email. If you'd like to stay up to date, you can enable these emails in your settings.`
+
+*Display if user logged in:*
+- Button: `Enable email updates`
+- Consent: `By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`
+
+## Unsubscribe page
+
+Legal regulations (e.g. CAN-SPAM act) have specific requirements to include an unsubscribe link in commercial email messages to manage email preferences without requiring a login.
+
+As a user I can:
+- Access my unique/personal unsubscribe page through a link without logging in. The link would typically be included with non-transactional emails from Pontoon as an “Unsubscribe” link in the footer.
+  - Link technical specs:
+  1) Pontoon generates a random unique identifier and assigns it to each user. The algorithm doesn't use any existing user data to generate the ID, which needs to be sufficiently complex to avoid a potential attacker guessing.
+  2) The unsubscribe page can be accessed without login. If a user ID is not provided, or it's incorrect, it will display an error message.
+  3) Unsubscribe links in emails will include the user's unique identifier in the URL to correctly identify the user.
+- All options are managed from a single page; I do not need to navigate from the page linked in the email to manage subscriptions.
+- Unsubscribe from all emails with a single button.
+- Subscribe for emails with a single button.
+- Receive a confirmation that I have successfully unsubscribed/subscribed. I am also instructed how to change my subscriptions later through my user preferences.
```

Comment:
I think we can drop all of this, because it is presented with more detail below. Before I saw the actual copy below, I was confused by these bullet points (e.g. why would I want to subscribe on the unsubscribe page, how exactly am I instructed, etc.).